### DISCUSSION TITLE
What are the differences between this extension and the remote-containers extension?

### DISCUSSION THREAD

**Discussion by abceleung** [microsoft/vscode-docker#3245] [Q&A]
Interestingly, both are created by Microsoft.

Are they supposed to be used together?

**Comment by bwateratmsft** [comment]
If you're referring to the [Docker extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker), then this and that are one and the same, because this is the repo for that extension.

**Reply by abceleung** [reply]
Oops, I meant the _Remote - Containers_ extension.
https://github.com/Microsoft/vscode-remote-release

**Reply by bwateratmsft** [reply]
Gotcha. Remote - Containers is used for attaching VSCode to containers via VSCode remoting. You can read more about it here: https://code.visualstudio.com/docs/remote/containers
